apps/utils/lib/utils/json_rpc/encoding.ex

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-defmodule OMG.RPC.Web.Encoding do
+defmodule Utils.JsonRPC.Encoding do
```

Comment:
the `JsonRPC` part itself - we're not using JSONRPC (the protocol), so maybe lets call it `HttpRPC` to avoid confusion (to add to it - we used to use JSONRPC and then changed to HTTP-RPC, so putting the word `Json` back in might be extra confusing)